concent_api/concent_api/settings/development.py

```diff
@@ -48,3 +48,5 @@
 MIDDLEMAN_PORT = DEFAULT_INTERNAL_PORT
 
 SIGNING_SERVICE_PUBLIC_KEY = b'\x19\xaew&P>\xce\xd7D\xfb\xbff)55\xc1%\xea\xebN\x0e\x05>\xf7v\x0c\x94I\xe4\xa3\x80\x1dl\xed\xb5j\xe9apZ\x8c\x86\xe2\xb2x^M}>8c\x90\x0e\xa2\xf4\x8cOb\xb1\x1er\x11\xd7\xee'
+
+GNT_DEPOSIT = '0xcfB81A6EE3ae6aD4Ac59ddD21fB4589055c13DaD'
```

Comment:
I believe this is some fake value ? :)

Reply:
No, it's our contract address that we use on `dev` cluster. It's not some secret value like private key that someone can use to take advantage of.